docs/advtool-installation.html

```diff
@@ -62,109 +62,118 @@ <h4 class="ibm-h4">Taking advantage of leading edge tools and libraries</h4>
   <thead>
     <tr>
       <th></th>
+      <th>Advance Toolchain 14.0</th>
       <th>Advance Toolchain 13.0</th>
       <th>Advance Toolchain 12.0</th>
-      <th>Advance Toolchain 11.0</th>
```

Comment:
Are we dropping AT11 support?  If so, that might be nice to add to the commit message? Or, in a separate commit? Not a big deal, really.

Reply:
> Are we dropping AT11 support? 

We are.

> If so, that might be nice to add to the commit message? Or, in a separate commit?

You're right, much more cleaner.
I update the PR, a separate commit updates the documentation to remove AT 11.0.